<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleEarthView</class>
 <widget class="QMainWindow" name="GoogleEarthView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="BtnExpand">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Zoom In</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/expand.bmp</normaloff>:/buttons/expand.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnHeading">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Heading Up</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/headup.bmp</normaloff>:/buttons/headup.bmp</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnFixCent">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Fix Plot Center</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/point.bmp</normaloff>:/buttons/point.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnGETilt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>136</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Perspective View</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/tilt.bmp</normaloff>:/buttons/tilt.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnOpt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Show Objects</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/navctr.bmp</normaloff>:/buttons/navctr.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnGENorm">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Objective View</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/norm.bmp</normaloff>:/buttons/norm.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnRotL">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Rotate Left</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/rotl.bmp</normaloff>:/buttons/rotl.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnEnaAlt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enable Altitude</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="rtkplot_qt.qrc">
      <normaloff>:/buttons/alt.bmp</normaloff>:/buttons/alt.bmp</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnShrink">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>0</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Zoom Out</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/shrink.bmp</normaloff>:/buttons/shrink.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="BtnRotR">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>30</width>
      <height>32</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Rotate Right</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
      <normaloff>:/buttons/rotr.bmp</normaloff>:/buttons/rotr.bmp</iconset>
    </property>
    <property name="flat" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Debug">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>0</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Panel3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>9</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="Panel5" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>119</width>
      <height>45</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" stdset="0">
     <string>Panel5</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QPushButton" name="BtnClose">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="Panel8" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>348</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="BtnOpt1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Layer Borders</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/navctr.bmp</normaloff>:/buttons/navctr.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show Navigation Control</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/grid.bmp</normaloff>:/buttons/grid.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show Scale Legend</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/scale.bmp</normaloff>:/buttons/scale.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show Overview Map</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/vmap.bmp</normaloff>:/buttons/vmap.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show Lat/Long Grid</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/status.bmp</normaloff>:/buttons/status.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show Status Bar</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/undu.bmp</normaloff>:/buttons/undu.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Layer Terrain</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/road.bmp</normaloff>:/buttons/road.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Layer Loads</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/building.bmp</normaloff>:/buttons/building.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BtnOpt9">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Layer Buildings</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
         <normaloff>:/buttons/border.bmp</normaloff>:/buttons/border.bmp</iconset>
       </property>
       <property name="flat" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../rtkconv_qt/rtkconv_qt.qrc"/>
  <include location="rtkplot_qt.qrc"/>
 </resources>
 <connections/>
</ui>
